docs/api/main-config/main-config-static-dirs.mdx

```diff
@@ -17,6 +17,10 @@ Sets a list of directories of [static files](../../configure/integration/images-
 
 {/* prettier-ignore-end */}
 
+<Callout variant="info">
+When using Vite-based frameworks, additional directories may be copied to your build directory because of Vite's own [static asset handling](https://vite.dev/guide/assets#the-public-directory). You can set Vite's `publicDir` option to `false` to disable this behavior.
+</Callout>
```

Comment:
@Sidnioulz, when you're able, can you add the same callout in https://storybook.js.org/docs/configure/integration/images-and-assets#serving-static-files-via-storybook-configuration for a bit more visibility so that this information doesn't stay "buried" in the configuration docs?